earth2studio/models/dx/cbottle_sr.py

```diff
@@ -101,130 +110,100 @@ class CBottleSR(torch.nn.Module, AutoModelMixin):
 
     Parameters
     ----------
-    core_model : torch.nn.Module
-        Core pytorch model implementing the diffusion-based super-resolution
+    sr_model : SuperResolutionModel
+        Core cBottle super-resolution helper implementing the diffusion process
     lat_lon : bool, optional, by default True
         Lat/lon toggle, if true the model will expect a lat/lon grid as input and output a lat/lon
         grid. If false, the native nested HealPix grid will be used for input and output.
-        Input HEALPix is level 6 and output HEALPix is level 10 with NEST pixel ordering.
+        Input HEALPix is level 6 and output HEALPix is level 10 with NEST pixel ordering
     output_resolution : Tuple[int, int], optional
         High-resolution output dimensions for lat/lon output. Only used when
         lat_lon=True, by default (2161, 4320)
-    super_resolution_window : Union[None, Tuple[int, int, int, int]], optional
-        Super-resolution window. If None, super-resolution is done
-        on the entire global grid. If provided, the super-resolution window is a tuple
-        of (lat_south, lon_west, lat_north, lon_east) and will only apply super-resolution
-        to the specified window. For lat/lon output, the result will just be returned for
-        the specified window with the specified output resolution.
-    sampler_steps : int, optional
```

Comment:
Sampler steps should stay and remain controllable on the fly after the model is loaded... every forward pass you should set the num steps of the superres model to this